src/pybind/cephfs.py

```diff
@@ -82,6 +83,12 @@ class cephfs_statvfs(Structure):
                 ("f_flag", c_uint),
                 ("f_namemax", c_uint)]
 
+class cephfs_dirent(Structure):
+    _fields_ = [("d_ino", c_long),
+                ("d_off", c_ulong),
+                ("d_reclen", c_ushort),
+                ("d_type", c_uint8),
+                ("d_name", c_char*256)]
```

Comment:
Actually I looked at Client::fill_dirent. Is this make sense? @liewegas @gregsfortytwo